cli/registry/client/client.go

```diff
@@ -121,7 +121,10 @@ func (c *client) PutManifest(ctx context.Context, ref reference.Named, manifest
 	}
 
 	dgst, err := manifestService.Put(ctx, manifest, opts...)
-	return dgst, errors.Wrapf(err, "failed to put manifest %s", ref)
+	if err != nil {
+		return dgst, errors.Wrapf(err, "failed to put manifest %s", ref)
```

Comment:
same here, is `dgst` set on error?

Reply:
Yeah, honestly, same situation as my other comment; we should _probably_ return a `nil` or "zero" value here, because results should (almost) never be consumed if there's an error, but this was also a case where I  would have to check all possible implementations; `manifestService.Get` is an interface, so now the fun starts to check which one we actually use, and how that one behaves 🙈 

So I was taking the "safe" side and kept it as-is, leaving that for a future exercise 😅 

<img width="957" alt="Screenshot 2025-02-21 at 10 33 31" src="https://github.com/user-attachments/assets/ab0be332-516b-4b39-9e9e-c07a90dcebf9" />